util/src/kvdb.rs

```diff
@@ -598,4 +651,12 @@ mod tests {
 		let _ = Database::open_default(path.as_path().to_str().unwrap()).unwrap();
 		test_db(&DatabaseConfig::default());
 	}
+
+	#[test]
+	fn auto_compaction() {
```

Comment:
i'd imagine a test named `auto_compaction` would eventually be cross-platform enough to merit different versions for different platforms.